test/test_source_base.py

```diff
@@ -540,3 +542,24 @@ def testImageBytes():
     assert 'ImageBytes' in repr(ib)
     assert ib._repr_jpeg_() is None
     assert ib._repr_png_() is None
+
+
+def testOutputFormats():
+    imagePath = datastore.fetch('sample_image.ptif')
+    testDir = os.path.dirname(os.path.realpath(__file__))
+    imagePathRGBA = os.path.join(testDir, 'test_files', 'rgba_geotiff.tiff')
+    for format in large_image.constants.TileOutputMimeTypes:
+        if format == 'TILED':
+            continue
```

Comment:
I think it'd be better to parameterize this with pytest so that if a subset of formats fail, we can capture that

Reply:
Agreed.  See b327a3d470dc78cc5a16e60026d99bb2d12d044c